### PR TITLE
[DOCS] ARM CPU plugin docs

### PR DESCRIPTION
**Pull request by alvoron**
TODO:
- [x] get disclaimer wording from Matthew and add it to device page
- [x] update the list of supported layers

### CODE REVIEW

File: docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md [hunk 1]
```diff
@@ -38,17 +40,17 @@ Devices similar to the ones we have used for benchmarking can be accessed using
...
-| [Models caching](../Model_caching_overview.md) | Yes | Partial | ? | ? |
-| [Dynamic shapes](../ov_dynamic_shapes.md) | Yes | Partial | ? | ? |
-| Import/Export | Yes | No | ? | ? |
-| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? |
-| [Stateful models](../network_state_intro.md) | Yes | No | ? | ? |
-| [Extensibility](@ref openvino_docs_Extensibility_UG_Intro) | Yes | Yes | ? | ? |
+| Capability | CPU | [GPU](./GPU.md) | GNA | VPU | [ARM CPU](ARM_CPU.md) |
+| ---------- | --- | --- | --- | --- | --- |
+| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? | No |
+| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? | No |
```
Comment (terfendail) [inline docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md:46]: As far as I know multi-device execution spread independent simultaneously executed infer requests between different devices so should be supported by Arm CPU plugin

File: docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+[Hello Query Device C++ Sample](../../../samples/cpp/hello_query_device/README.md) can be used to print out supported data types for all detected devices.
+
+
+## Supported properties
+The plugin supports the properties listed below.
+
+### Read-write properties
+All parameters must be set before calling `ov::Core::compile_model()` in order to take effect or passed as additional argument to `ov::Core::compile_model()`
+
+- ov::enable_profiling
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md:33]: I suppose you also support streams, pinning, etc
@apankratovantonp  please, provide a full list of properties.

File: docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md [hunk 3]
```diff
@@ -38,17 +40,17 @@ Devices similar to the ones we have used for benchmarking can be accessed using
...
+| Capability | CPU | [GPU](./GPU.md) | GNA | VPU | [Arm&reg; CPU](ARM_CPU.md) |
+| ---------- | --- | --- | --- | --- | --- |
+| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? | No |
+| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? | No |
+| [Automatic batching](../automatic_batching.md) | No | Yes | ? | ? | No |
+| [Multi-stream execution](@ref openvino_docs_optimization_guide_dldt_optimization_guide) | Yes | Yes | ? | ? | Yes |
+| [Models caching](../Model_caching_overview.md) | Yes | Partial | ? | ? | No |
+| [Dynamic shapes](../ov_dynamic_shapes.md) | Yes | Partial | ? | ? | No |
+| Import/Export | Yes | No | ? | ? | No |
+| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? | No |
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md:52]: do we support any of convert, transpose, interpolate, color space conversions? We probably need to use partial and give more explanations in the plugin page.

File: docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md [hunk 4]
```diff
@@ -38,17 +40,17 @@ Devices similar to the ones we have used for benchmarking can be accessed using
...
-| Import/Export | Yes | No | ? | ? |
-| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? |
-| [Stateful models](../network_state_intro.md) | Yes | No | ? | ? |
-| [Extensibility](@ref openvino_docs_Extensibility_UG_Intro) | Yes | Yes | ? | ? |
+| Capability | CPU | [GPU](./GPU.md) | GNA | VPU | [Arm&reg; CPU](ARM_CPU.md) |
+| ---------- | --- | --- | --- | --- | --- |
+| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? | No |
+| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? | No |
+| [Automatic batching](../automatic_batching.md) | No | Yes | ? | ? | No |
+| [Multi-stream execution](@ref openvino_docs_optimization_guide_dldt_optimization_guide) | Yes | Yes | ? | ? | Yes |
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md:48]: I suppose MULTI, HETERO, multi stream executions are supported.

Comment (alvoron) [reply docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md:48]: @apankratovantonp could you please comment about MULTI and HETERO?

File: docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md (hunk 2 above)
Comment (apankratovantonp) [reply docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md:54]: Thus these options are in the supported list they do not provide expected threading control. We even test only in latency (sync) mode only. I am going to enable tbb threading backend and provide full threading control as in the CPU plugin. It won't give any valuable throughput gain but give full threading control. So we can claim support of these options.

File: docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md (hunk 1 above)
Comment (apankratovantonp) [reply docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md:46]: Yes we support work under multi and hetero plugin

File: docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md [hunk 5]
```diff
@@ -121,155 +129,157 @@ For setting relevant configuration, refer to the
...
+| Acosh                          | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*|Supported|
+| Activation-Clamp               | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Activation-ELU                 | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Activation-Exp                 | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Activation-Leaky ReLU          | Supported     |Supported\*\*\*| Supported     | Supported     | ?     | Supported     |
+| Activation-Not                 | Supported     |Supported\*\*\*| Supported     | Not Supported | ?     | Supported     |
+| Activation-PReLU               | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Activation-ReLU                | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Activation-ReLU6               | Supported     |Supported\*\*\*| Supported     | Not Supported | ?     | Supported     |
```
Comment (terfendail) [inline docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md:144]: There is no spec for ReLU-6

File: docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md [hunk 6]
```diff
@@ -121,155 +129,157 @@ For setting relevant configuration, refer to the
...
+| Convolution-Grouped            | Supported     | Supported     | Supported     | Not Supported | ?     | Supported     |
+| Convolution-Grouped 3D         | Supported     | Supported     | Not Supported | Not Supported | ? | Not Supported |
+| Convolution-Ordinary           | Supported     | Supported     | Supported     | Supported\*   | ?     | Supported     |
+| Convolution-Ordinary 3D        | Supported     | Supported     | Not Supported | Not Supported | ? | Not Supported |
+| Cos                            | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Cosh                           | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Crop                           | Supported     | Supported     | Supported     | Supported     | ?     | Supported     |
+| CTCGreedyDecoder               | Supported\*\* | Supported\*\* | Supported\*   | Not Supported |Supported\*\*\*\*| Supported     |
+| Deconvolution                  | Supported     | Supported     | Supported     | Not Supported | ? | Supported     |
```
Comment (terfendail) [inline docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md:168]: There are no specs for Deconvolution and Crop.
However I've found mention of a Crop layer in the [issue](https://github.com/openvinotoolkit/openvino/issues/27) created in 2018

File: docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md [hunk 7]
```diff
@@ -121,155 +129,157 @@ For setting relevant configuration, refer to the
...
+| Eltwise-LogicalAnd             | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-LogicalOr              | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-LogicalXor             | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-Max                    | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-Min                    | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-Mul                    | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Eltwise-NotEqual               | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported\*   | Supported     |
+| Eltwise-Pow                    | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-Prod                   | Supported     |Supported\*\*\*| Supported     | Supported     | ?     | Supported     |
```
Comment (terfendail) [inline docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md:190]: There is no spec for Eltwise-Prod

File: docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md [hunk 8]
```diff
@@ -121,155 +129,157 @@ For setting relevant configuration, refer to the
...
+| Eltwise-SquaredDiff            | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported     | Supported     |
+| Eltwise-Sub                    | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Eltwise-Sum                    | Supported     |Supported\*\*\*| Supported     | Supported     | ?     | Supported     |
```
Comment (terfendail) [inline docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md:193]: There is no spec for Eltwise-Sum. Should it be considered CumSum layer?

**Review by alalek — COMMENTED** [review]
Need to align:
> `The ARM&reg; CPU plugin`
> `The Arm&reg; CPU plugin`

File: docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+The set of supported layers is defined on [Operation set specification](https://github.com/openvinotoolkit/openvino_contrib/wiki/ARM-plugin-operation-set-specification).
+
+
+## Supported inference data types
+The Arm&reg; CPU plugin supports the following data types as inference precision of internal primitives:
+
+- Floating-point data types:
+  - f32
+  - f16
```
[the remark this answers is not in the document]
Comment (alvoron) [reply docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md:17]: I set f32 everywhere

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Ported as a part of https://github.com/openvinotoolkit/openvino/pull/11040, please, check that everything is correct

**Pull request merged**